packages/integrations/cloudflare/runtime.d.ts

```diff
@@ -0,0 +1,8 @@
+export type {
+  WorkerRuntime,
+  PagesRuntime
+} from './dist/runtime';
+
+export {
+  getRuntime
+} from './dist/runtime';
```

Comment:
```suggestion
} from './dist/runtime.js';

export {
  getRuntime
} from './dist/runtime.js';
```

I believe these should end with `.js` for node16 resolution support. We also have the same pattern for the main Astro types.

Reply:
Doesn't look like we do: https://github.com/withastro/astro/blob/main/packages/astro/types.d.ts#L1-L2

Reply:
Ah looks like we're only doing it for ambient types (<https://github.com/withastro/astro/blob/0e163533ea8a9800b9e288fd0ae6853ace878101/packages/astro/astro-jsx.d.ts#L26-L32>). Maybe it's fine to not have it for the module files, looks like it's also working today.